recipes/jsonrpcserver/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "jsonrpcserver" %}
+{% set version = "4.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 240c517f49b0fdd3bfa428c9a7cc581126a0c43eca60d29762da124017d9d9f4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - jsonrpcserver
+
+about:
+  home: https://github.com/bcb/jsonrpcserver
+  license: MIT
+  license_family: MIT
+  license_file: https://github.com/bcb/jsonrpcserver/blob/master/LICENSE
```

The license here should be just ``LICENSE`` as it is being packaged on pypi and also on github tarballs
```suggestion
  license_file: LICENSE
```

---

Fixed.